packages/browser/src/Session.ts

```diff
@@ -424,4 +431,17 @@ export class Session extends EventEmitter {
   onSessionRestore(callback: (currentUrl: string) => unknown): void {
     this.on("sessionRestore", callback);
   }
+
+  private setSessionInfo(
+    sessionInfo: ISessionInfo & { isLoggedIn: true }
+  ): void {
+    this.info.isLoggedIn = sessionInfo.isLoggedIn;
+    this.info.webId = sessionInfo.webId;
+    this.info.sessionId = sessionInfo.sessionId;
+    if (typeof sessionInfo.expirationDate === "number") {
```

Comment:
```suggestion
    this.info.sessionId = sessionInfo.sessionId;
    this.info.expirationDate = sessionInfo.expirationDate;
    if (typeof sessionInfo.expirationDate === "number") {
```
This will allow, in an upcoming ticket, to change the behaviour of the callback set by `setTimeout` to checking if the current expiration date (`this.info.expirationDate`) is close to `Date.now()` (with a certain tolerance). This way, if silent authentication happens in an `iframe`, the callback will not log the session out.

Reply:
Whoops yeah, that's kinda important: 4134dc9f